assignment.js

```diff
@@ -32,6 +32,14 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+let result =''
+let j = 2
+
+for(let i=0;i<=12;i++){
+    result += (i * j)+ ' ';
```

Comment:
@Defy1x be careful with indentation..this should only be indented 1 space, this currently has 2 levels of indentation